Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/AdaptiveHoughTransformSeeder.hpp

```diff
@@ -0,0 +1,478 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// @file AdaptiveHoughTransformSeeder.hpp
+// @author Tomasz Bold
+// @brief Implements track-seeding using Adaptive Hough Transform.
+//
+
+#pragma once
+
+#include "Acts/EventData/SourceLink.hpp"
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Utilities/Delegate.hpp"
+#include "Acts/Utilities/Grid.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "Acts/Utilities/Result.hpp"
+#include "ActsExamples/EventData/Index.hpp"
+#include "ActsExamples/EventData/Measurement.hpp"
+#include "ActsExamples/EventData/ProtoTrack.hpp"
+#include "ActsExamples/EventData/SimSeed.hpp"
+#include "ActsExamples/EventData/SimSpacePoint.hpp"
+#include "ActsExamples/Framework/DataHandle.hpp"
+#include "ActsExamples/Framework/IAlgorithm.hpp"
+#include "ActsExamples/Framework/ProcessCode.hpp"
+
+#include <cstddef>
+#include <list>
+#include <memory>
+#include <numbers>
+#include <string>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+namespace ActsExamples {
+struct AlgorithmContext;
+}
+
+namespace Acts {
+class TrackingGeometry;
+}
+
+namespace ActsExamples {
+// Helper class describing one section of the accumulator space
+class AccumulatorSection {
```

Comment:
Maybe this could use a more specific name. `AccumulatorSection` sounds pretty generic. You could consider moving this inside the Hough Algorithm. If it's not used anywhere else, you could also move it to the `.cpp` file into an anonymous namespace.

Reply:
@paulgessinger - I have addressed the easy ones. I wonder if this one could ba addressed after the next round when we'll move this "core" code.